### PR TITLE
[MINOR][DOC] Correct some document description errors

### PR DESCRIPTION
**Pull request by 10110346**
## What changes were proposed in this pull request?

Correct some document description errors.

## How was this patch tested?
N/A

### CODE REVIEW

**Comment by 10110346** [comment]
retest this please

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 1]
```diff
@@ -430,8 +430,8 @@ package object config {
       .doc("The chunk size in bytes during writing out the bytes of ChunkedByteBuffer.")
       .bytesConf(ByteUnit.BYTE)
       .checkValue(_ <= ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH,
-        "The chunk size during writing out the bytes of" +
-        " ChunkedByteBuffer should not larger than Int.MaxValue - 15.")
+        "The chunk size during writing out the bytes of ChunkedByteBuffer should" +
+          s" not larger than ${ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH}.")
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: not larger than => be at most
or => not be greater than

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 2]
```diff
@@ -503,7 +503,7 @@ package object config {
         "made in creating intermediate shuffle files.")
       .bytesConf(ByteUnit.KiB)
       .checkValue(v => v > 0 && v <= ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH / 1024,
-        s"The file buffer size must be greater than 0 and less than" +
+        s"The file buffer size must be greater than 0 and not greater than" +
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: You can say positive instead of 'greater than 0' if you like, as above

**Review by srowen — CHANGES_REQUESTED** [review]

File: core/src/main/scala/org/apache/spark/internal/config/package.scala (hunk 1 above)
Comment (10110346) [reply core/src/main/scala/org/apache/spark/internal/config/package.scala]: ok, thanks

**Review by srowen — APPROVED** [review]

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 3]
```diff
@@ -513,7 +513,7 @@ package object config {
         "is written in unsafe shuffle writer. In KiB unless otherwise specified.")
       .bytesConf(ByteUnit.KiB)
       .checkValue(v => v > 0 && v <= ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH / 1024,
-        s"The buffer size must be greater than 0 and less than" +
+        s"The buffer size must be positive and not greater than" +
```
Comment (HyukjinKwon) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: not greater than -> less than or equal to

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 4]
```diff
@@ -503,7 +503,7 @@ package object config {
         "made in creating intermediate shuffle files.")
       .bytesConf(ByteUnit.KiB)
       .checkValue(v => v > 0 && v <= ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH / 1024,
-        s"The file buffer size must be greater than 0 and less than" +
+        s"The file buffer size must be positive and not greater than" +
```
Comment (HyukjinKwon) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: not greater than -> less than or equal to

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 5]
```diff
@@ -430,8 +430,8 @@ package object config {
       .doc("The chunk size in bytes during writing out the bytes of ChunkedByteBuffer.")
       .bytesConf(ByteUnit.BYTE)
       .checkValue(_ <= ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH,
-        "The chunk size during writing out the bytes of" +
-        " ChunkedByteBuffer should not larger than Int.MaxValue - 15.")
+        "The chunk size during writing out the bytes of ChunkedByteBuffer should" +
+          s" not be greater than ${ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH}.")
```
Comment (HyukjinKwon) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: not be greater than -> less than or equal to

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Few nit comments because I thought we should avoid: negative comparison; however, let me leave it to @srowen.

**Comment by 10110346** [comment]
retest this please

File: core/src/main/scala/org/apache/spark/internal/config/package.scala (hunk 3 above)
Comment (srowen) [reply core/src/main/scala/org/apache/spark/internal/config/package.scala]: Yeah, I could go either way on those. I wouldn't mind standardizing on "less than or equal to", sure. @10110346 would you mind taking one more pass accordingly?

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**